app/pricing/FormattersComponents.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+
+import * as React from "react";
+
+const SERVER_LOCAL = "en-US";
+
+const IntlNumberFormat = ({
+  options,
+  serverLocal = SERVER_LOCAL,
+  value,
+}: {
+  options: Intl.NumberFormatOptions;
+  serverLocal?: string;
+  value: number;
+}) => {
+  const optionsRef = React.useRef(options);
+  const [formatter, setFormatter] = React.useState(
+    new Intl.NumberFormat(serverLocal, optionsRef.current),
+  );
+
+  React.useEffect(() => {
+    setFormatter(new Intl.NumberFormat(undefined, optionsRef.current));
```

Comment:
It would be much more efficient to store value in state because if locale produces the same it will avoid a re-render.

Reply:
I don't understand how to do that and I don't want to delay the PR. As the efficiency improvement has no value here, I'll ignore it.